docs/dev/api/rdc-access-api/integration-guide.md

```diff
@@ -0,0 +1,343 @@
+
+- `descriptor`: Static identifier (for example, `iPhone_13_real`) consistent across the API.
+- `deviceName`: Descriptive label you see in the UI (for example, `iPhone 13 real private`).
+
+The `deviceName` query parameter accepts an exact value or regular expression and matches against both `descriptor` and `deviceName`. This holds true for every endpoint that takes `deviceName`.
+
+> Need the full schema for these fields? See `Device` and `Session` definitions in the [Access API spec](/real-device-access-api).
+
+## Quick Start Examples
+
+### Check Device Status
+You can retrieve a list of all available real devices and filter them based on various criteria.
+
+#### List All Devices
+```shell
+curl -X GET \
+  -u $AUTH \
+  "$BASE_URL/devices/status"
+```
+
+#### Device Filtering Options
+The `/devices/status` endpoint supports the following query parameters:
+
+- `state`: Filter by device state (`AVAILABLE`, `IN_USE`, `CLEANING`, `REBOOTING`, `MAINTENANCE`, `OFFLINE`).
+- `privateOnly`: Set to `true` to show only your account's private devices.
+- `deviceName`: Filter by descriptor or friendly name. Accepts regular expressions (for example, `iPhone.*`).
+
+#### Examples:
+```shell
+# Get all devices
+curl -X GET -u $AUTH "$BASE_URL/devices/status"
+
+# Filter by device state
+curl -X GET -u $AUTH "$BASE_URL/devices/status?state=AVAILABLE"
+
+# Show only private devices
+curl -X GET -u $AUTH \
+  "$BASE_URL/devices/status?privateOnly=true"
+
+# Filter by device identifier (supports regular expression patterns)
+curl -X GET -u $AUTH \
+  "$BASE_URL/devices/status?deviceName=iPhone.*"
+
+# Combine multiple filters
+curl -X GET -u $AUTH \
+  "$BASE_URL/devices/status?state=AVAILABLE&privateOnly=true&deviceName=iPhone.*"
+```
+
+#### Example Response
+
+```json
+{
+  "devices": [
+    {
+      "descriptor": "iPhone_13_real",
+      "isPrivateDevice": true,
+      "status": "AVAILABLE"
+    },
+    {
+      "descriptor": "Samsung_Galaxy_S21_real",
+      "isPrivateDevice": false,
+      "status": "IN_USE",
+      "inUseBy": [
+        {
+          "username": "john.smith"
+        }
+      ]
+    }
+  ]
+}
+```
+
+### Create a Device Session
+Start a new session with a `POST` request to `/sessions`.
+> ***Note:*** The `deviceName` and `os` parameters are optional. If omitted, Sauce Labs selects an available device automatically. You can also include the optional `configuration` object from the API spec to set `sessionDuration` (ISO-8601) or attach a Sauce Connect `tunnel`.
+
+#### Example
+```shell
+curl -X POST -u $AUTH \
+  -H "Content-Type: application/json" \
+  -d '{
+    "device": {
+      "deviceName": "iPhone_16_real",
+      "os": "ios"
+    }
+  }' \
+  "$BASE_URL/sessions"
+```
+
+#### Example Response
+
+```json
+{
+  "id": "{session_id}",
+  "state": "PENDING",
+  "links": null,
+  "device": null,
+  "error": null
+}
+```
+
+### Session Lifecycle Cheat Sheet
```

Comment:
How is this a cheat sheet? 

A cheat sheet is usually a collection of examples of common use cases.